### ISSUE TITLE
hide action button

### ISSUE THREAD

**Issue by syrakozz** [lourenco-marinho/ActionButton#22]
can you plz add the ability to hide the action button compleetlty

**Comment by sagarscript** [comment]
I Agree Big Time, I have my action button that SHOULD only be visible to logged in users, guests shouldn't be able to see it as they cant use it. As a work around my action checks to see if the user is logged in before `button.toggleMenu()`

... I have checked the source code and have found that there are two ways to fix this very quickly...

Manually change 
```
    /// The button that will be presented to the user
    fileprivate var floatButton: UIButton!
```
```
    /// The button that will be presented to the user
    public var floatButton: UIButton!
```

this would allow you to hide the button like so:
`actionButton.floatButton.isHidden = true`
and show the button:
`actionButton.floatButton.isHidden = false`

Although when updating pods you would have to re-do this step each time...

A better solution would be for the developer to amend the code to include:

```
public var isHidden: Bool {
        willSet(newIsHidden) {
            self.isHidden = newIsHidden
        }
        get{
           return self.isHidden 
       }
}
```

I hope this helps users who want to hide/show and also for the developer to easily add the functionality to the pod
